msmbuilder/tpt/mfpt.py

```diff
@@ -26,7 +26,45 @@
 __all__ = ['mfpts']
 
 
-def mfpts(msm, sinks=None, lag_time=1.):
+def create_perturb_params(tmat, countsmat):
+    '''
+    Helper function for computing mfpts when "errors" argument is passed. 
+    Generates the means and standard deviations of the Gaussian random variables
+    for each transition probability that will be sampled from during the error calculation.
+    
+    Parameters:
+    ----------
+    tmat: np.ndarray
+        The msm transition matrix 
+    countsmat: np.ndarray
+        The msm counts matrix
+    '''
+    counts = (np.ones((len(tmat), len(tmat))) * np.sum(countsmat, axis=1)).transpose()
+    scale = ((tmat - tmat ** 2) / counts ** 0.5) + 10 ** -15
+    return (tmat, scale)
+
+
+def perturb_tmat(loc, scale):
+    '''
+    Helper function for computing mfpts when "errors" argument is passed.
+    Perturbs each nonzero transition probability by treating it as a Gaussian random variable
+    and sampling from it once.
+    
+    Parameters:
+    ----------
+    loc: np.ndarray:
+        The transition matrix, whose elements serve as the means of the Gaussian random variables
+    scale: np.ndarray:
+        The matrix of standard deviations of the Gaussians. For transition probability (i,j), this is 
+        assumed to be the standard error of the mean of a binomial distribution with p = transition probability 
+        and number of observations equal to the summed counts in row i.
+    '''
+    output = np.vectorize(np.random.normal)(loc, scale)
+    output[np.where(output < 0)] = 0
+    return (output.transpose() / np.sum(output, axis=1)).transpose()
+
+
+def mfpts(msm, sinks=None, lag_time=1., errors='-1'):
```

Comment:
Yeah that's a good point. I could add another kwarg if we want to set a default value for n_samples. Also, the '-1' is just supposed to represent "False", since that's how I've seen it done in a lot of old msmbuilder 2 code, but that's probably a bit confusing. I'll change it.